src/app/clinic/clinic_facade.cpp

```diff
@@ -30,35 +30,45 @@ void Clinic::appendCalendar(const std::shared_ptr<Calendar>& calendar)
     calendar_ = calendar;
 }
```

Comment:
I think that we should change name to setCalendar as "append" is misleading.

I mean, we have just 1 calendar which we set every time the function is called.
We do not add another calendar each time, as the name suggests.